Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -1364,58 +1364,51 @@ def search_domain_command(args):
-        )
-
-        autofocus_domain_output = parse_indicator_response(indicator, raw_tags, indicator_type)
 
-        # create human readable markdown for ip
-        tags = autofocus_domain_output.get('Tags')
-        table_name = f'{VENDOR_NAME} {indicator_type} reputation for: {domain_name}'
-        if tags:
-            indicators_data = autofocus_domain_output.copy()
-            del indicators_data['Tags']
-            md = tableToMarkdown(table_name, indicators_data)
-            md += tableToMarkdown('Indicator Tags:', tags)
+        if indicator:
+            raw_tags = raw_res.get('tags')
+            score = calculate_dbot_score(indicator, indicator_type)
+            dbot_score = Common.DBotScore(
+                indicator=domain_name,
+                indicator_type=DBotScoreType.DOMAIN,
+                integration_name=VENDOR_NAME,
+                score=score
+            )
+            domain = Common.Domain(
+                domain=domain_name,
+                dbot_score=dbot_score,
+                creation_date=indicator.get('whoisDomainCreationDate'),
+                expiration_date=indicator.get('whoisDomainExpireDate'),
+                updated_date=indicator.get('whoisDomainUpdateDate'),
+                admin_email=indicator.get('whoisAdminEmail'),
+                admin_name=indicator.get('whoisAdminName'),
+                registrar_name=indicator.get('whoisRegistrar'),
+                registrant_name=indicator.get('whoisRegistrant')
+            )
+            autofocus_domain_output = parse_indicator_response(indicator, raw_tags, indicator_type)
+            # create human readable markdown for ip
+            tags = autofocus_domain_output.get('Tags')
+            table_name = f'{VENDOR_NAME} {indicator_type} reputation for: {domain_name}'
+            if tags:
+                indicators_data = autofocus_domain_output.copy()
+                del indicators_data['Tags']
+                md = tableToMarkdown(table_name, indicators_data)
+                md += tableToMarkdown('Indicator Tags:', tags)
+            else:
+                md = tableToMarkdown(table_name, autofocus_domain_output)
+            command_results.append(CommandResults(
+                outputs_prefix='AutoFocus.Domain',
+                outputs_key_field='IndicatorValue',
+                outputs=autofocus_domain_output,
+                readable_output=md,
+                raw_response=raw_res,
+                indicator=domain
+            ))
         else:
-            md = tableToMarkdown(table_name, autofocus_domain_output)
-
-        command_results.append(CommandResults(
-            outputs_prefix='AutoFocus.Domain',
-            outputs_key_field='IndicatorValue',
-            outputs=autofocus_domain_output,
-            readable_output=md,
-            raw_response=raw_res,
-            indicator=domain
-        ))
-
+            command_results.append(CommandResults(
```

Comment:
we should return dbot score 0 here

Reply:
Do we want to create the indicator too?

Reply:
yea the output should be the same for both cases, just in this one there wont be actual data